### ISSUE TITLE
OpenSSL WINCE 6.0 makefile modifications

### ISSUE THREAD

**Issue by mdeihly** [openssl/openssl#19633]
According to https://qiita.com/souju/items/94117c024862f57459c3 , it is recommended to modify the makefile generated by VS2008 to:
![image](https://user-images.githubusercontent.com/41584417/200779999-f3112f9a-2471-43e2-b33a-076ce57789a9.png)

However, I am using VS2005, so should I need to modify it to **MSC_VER=1300** or other version?

**Comment by aSoujyuTanaka** [comment]
No, you don't need it because of the changes of https://github.com/openssl/openssl/pull/11526,  as I wrote in the article,

**Comment by t8m** [comment]
I am sorry but WinCE is no longer supported. Closing.
